samples/net/eth_native_posix/README.rst

```diff
@@ -28,15 +28,24 @@ below.
    :compact:
 
 Normally one needs extra privileges to create and configure the TAP device in
-the host system. Use sudo to execute the Zephyr process with admin privileges,
-like this:
+the host system. If the user has set the
+:option:`CONFIG_ETH_NATIVE_POSIX_STARTUP_AUTOMATIC` option (this is disabled
+by default), then the user needs to use ``sudo`` to execute the Zephyr process
+with admin privileges, like this:
 
 .. code-block:: console
 
     sudo --preserve-env=ZEPHYR_BASE make run
 
 If the ``sudo --preserve-env=ZEPHYR_BASE`` gives an error,
-just use ``sudo --preserve-env`` instead:
+just use ``sudo --preserve-env`` instead.
+
+If that option is not enabled (this is the default), then the user should
+execute the ``net-setup.sh`` script from Zephyr `net-tools`_ repository.
+The script should be run before executing the Zephyr process. The script
+will create the zeth interface and setup IP addresses and routes.
+The manual setting does not require the Zephyr process to be run as a root
```

Comment:
```suggestion
By running ``net-setup.sh`` manually, you can run the Zephyr process as a non-root
```

Reply:
The irony is that net-setup.sh itself needs root. @jukkar's wording seems to be carefully crafted to blend that fact into background ;-). So, let me suggest my edition too after all:

~~~
While running ``net-setup.sh`` requires root access (once), afterwards 
Zephyr processes can be run as a non-root user.
~~~